doc/plugin_agent_workloadattestor_k8s.md

```diff
@@ -43,6 +49,7 @@ since [hostprocess](https://kubernetes.io/docs/tasks/configure-pod-container/cre
 | `token_path` | The path on disk to the bearer token used for kubelet authentication. Defaults to the service account token `/run/secrets/kubernetes.io/serviceaccount/token` |
 | `certificate_path` | The path on disk to client certificate used for kubelet authentication |
 | `private_key_path` | The path on disk to client key used for kubelet authentication |
+| `use_anonymous_authentication` | If true, use anonymous authentication for kubelet communication |
```

Comment:
maybe it is an overkill... but, what do you think about say again that kubelet must be started with `--anonymous-auth` in order to use this feature? I'm worried someone reading this fields think than just setting `use_anonymous_authentication` is enough. (maybe adding to agen_full.conf is enough)

Reply:
I thought about it, but we don't go into that level of detail for the other authentication-related configuration properties in this table, so I followed suit here and left the more detailed note above. I guess I would prefer to keep that consistent, and I feel like it's probably good enough to have the note above.